tools/find_regressions.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script for finding regressions fixed by some commit(s)
+# 
+# Usage:
+#     tools/find_regressions.sh [REVISION]
+# 
+# Compares output of DiffKemp between the current revision and REVISION.
+# If not specified, REVISION is equal to HEAD^.
+
+if [[ ! -d .git ]]; then
+    echo Run script from the repo root directory
+    exit 1
+fi
+
+function build() {
+    echo -n "Compiling $1-patch version ($(git rev-parse --short HEAD)), do not switch git branches..."
+    builddir=build-regression-$1
+    mkdir -p $builddir
+    cmake -B $builddir -S . -GNinja -DCMAKE_BUILD_TYPE=Debug > /dev/null
+    ninja -C $builddir > /dev/null
+    echo done
+}
+
+curref=$(git rev-parse --abbrev-ref HEAD)
+lastref=${1:-$curref^}
+
+build post
+git checkout -q $lastref
+build pre
+git checkout -q $curref
+
+logdir=logs/regression-$(date +%F-%T)
+mkdir -p $logdir
+
+versions=(80 147 193 240 305 348)
+
+for i in "${!versions[@]}"; do 
+    if [[ $(($i + 1)) == ${#versions[@]} ]]; then break; fi
+
+    old=linux-4.18.0-${versions[$i]}.el8
+    new=linux-4.18.0-${versions[$i + 1]}.el8
```

Comment:
An option to specify different versions of the kernel could be handy in the future, although it's not so important now and can be easily added.